### DISCUSSION TITLE
reset default display settings

### DISCUSSION THREAD

**Discussion by quadeqx** [dat-linux/community#48] [General]
I was familiarizing myself with this wonderful Linux distro and some display settings changed as shown in the attached image. This has caused title bars to be bigger than normal, the system info and also the data science control panel. Is there a way to default this. I've tried to run <dconf reset -f /> but nothing changed

![image](https://github.com/dat-linux/community/assets/154374993/6870027f-3d04-4cd7-aadf-7e1b48c3e0ee)

**Comment by k1consulting** [comment]
Maybe create a new user and login afresh.

**Comment by quadeqx** [comment]
This has not worked. I reinstalled the OS and the same happened again. Now it's even more worse
![dat](https://github.com/dat-linux/community/assets/154374993/910e0bcb-31c9-46f4-9615-e2bfaa5b6c1e)

**Reply by oss-linux-etc** [reply]
this does not look like the default desktop after install.
